qutebrowser/browser/webpage.py

```diff
@@ -218,6 +218,17 @@ def _ask(self, text, mode, default=None):
         q.deleteLater()
         return q.answer
 
+    def _show_pdfjs(self, reply):
+        """Show the reply with pdfjs."""
+        try:
+            page = pdfjs.generate_pdfjs_page(reply.url()).encode('utf-8')
+        except pdfjs.PDFJSNotFound:
+            page = (jinja.env.get_template('no_pdfjs.html')
+                    .render(url=reply.url().toDisplayString())
```

Comment:
The `pylint` error on this line is due to [a pylint bug](https://bitbucket.org/logilab/pylint/issues/490/jinja-templates-are-handled-as-str) - you can add something like this above:

```python
# pylint: disable=no-member
# WORKAROUND for https://bitbucket.org/logilab/pylint/issue/490/
```